src/common.props

```diff
@@ -3,9 +3,9 @@
     <PackageTags>akka;actors;actor model;Akka;concurrency;nlog</PackageTags>
     <Copyright>Copyright © 2013-2020 Akka.NET Contrib Team</Copyright>
     <Authors>Akka.NET Contrib Team</Authors>
-    <PackageReleaseNotes>Support structured logging and message templates using NLog 4.5</PackageReleaseNotes>
-    <VersionPrefix>1.3.3</VersionPrefix>
-    <PackageIconUrl>http://getakka.net/images/akkalogo.png</PackageIconUrl>
+    <PackageReleaseNotes>Dependency update for Akka 1.3.17 and NLog 4.5.11 for net452</PackageReleaseNotes>
+    <VersionPrefix>1.3.0</VersionPrefix>
```

Comment:
What i meant by this. Is only pointing out, that its the release_notes.md that controls the version that the nuget has. This version here, is only for building locally. ;)